docs/providers/amazon/aws/sensors/s3.rst

```diff
@@ -2,41 +2,62 @@ S3 Sensor Async
 """""""""""""""
 
 
-To waits for a key (a file-like instance on S3) to be present in a S3 bucket asynchronously
+To waits for a for one or multiple keys to be present in a S3 bucket asynchronously you can use
```

Comment:
This sentence is not grammatically correct.. `To waits for a for one or multiple keys`
Please take care while writing documentation

cc @kaxil

Reply:
Can we also double check how the sphinx documentation is coming up for this operator

Reply:
<img width="1171" alt="Screenshot 2022-08-12 at 5 16 57 PM" src="https://user-images.githubusercontent.com/92459020/184347828-a4e72475-e943-4196-8618-71b95ee9f6a9.png">

Reply:
Title need to be corrected to have `S3 Key Sensor Async` ?

Reply:
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/92459020/184348778-20ccc942-1f38-4362-8c53-315c75dadab3.png">

Done changed.

Reply:
Can you please consider changing first sentence to `Use S3KeySensorAsync to wait asynchronously for one or more keys to be available in an S3 bucket`

Reply:
Changed